<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qadastre_interface_form</class>
 <widget class="QDialog" name="qadastre_interface_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qadastre</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="title">
      <string>Interface QGIS</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Vous pouvez choisir d'appliquer une interface
simplifiée de QGIS pour consulter le cadastre
ou de revenir à l'interface par défaut</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QPushButton" name="btInterfaceQadastre">
          <property name="text">
           <string>Interface Qadastre</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btInterfaceQgis">
          <property name="text">
           <string>Interface QGIS</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
